### ISSUE TITLE
[DISCUSS] SeaTunnel RoadMap

### ISSUE THREAD

**Issue by CalvinKirs** [apache/seatunnel#2169]
Hello everyone, in the rapid development of the community, we have successively discussed major features such as unified API, core engine discussion (TBD), WebService & UI, so I will list them. If you have anything to add, you are welcome to discuss.

- 1: E2e(endpoint to endpoint) for connectors v2
- 2: Web Services #2134 
- 3: Support for multiple versions of the underlying engine (Spark 3.X & Flink 1.13.5+) #2142 
- 4: CataLog Design
- 5: CDC connector
- 6: Focussed on documentation 
- 7: More and more connectors #1946 
- 8: SeaTunnel Transform API #2678 
- 9: SeaTunnel Engine #2272 
- 10: CDC connectors #2394

**Comment by dik111** [comment]
Where can I find progress about spark 3.x support ?

**Comment by CalvinKirs** [comment]
> Where can I find progress about spark 3.x support ?

hi, here #2189

**Comment by itcrazy0918** [comment]
data quality : Synchronous data volume recording and comparison
metadata：Metadata is automatically discovered and configuration files are generated

**Comment by 2013650523** [comment]
CDC connector ,Is there a big man doing it?  I'm kind of interested in this, implementing the CDC connector on the new connector?

**Comment by CalvinKirs** [comment]
> CDC connector ,Is there a big man doing it? I'm kind of interested in this, implementing the CDC connector on the new connector?

Could you submit an issue calling on everyone to do it, you can start with a simple design.

**Comment by 2013650523** [comment]
> > CDC connector ,Is there a big man doing it? I'm kind of interested in this, implementing the CDC connector on the new connector?
> 
> Could you submit an issue calling on everyone to do it, you can start with a simple design.

Does the CDC connector here refer to the FLink CDC connector?  I don't quite understand. What does it look like?

**Comment by EricJoy2048** [comment]
> Does the CDC connector here refer to the FLink CDC connector? I don't quite understand. What does it look like?

No, The `CDC Connector` must use `SeaTunnel Connector API` instead of import `Flink CDC Connector` into SeaTunnel.

**Comment by melin** [comment]
支持限流，没有限流不是一个完整数据同步平台
